### PR TITLE
added soft deletable filter configuration

### PR DESCRIPTION
**Pull request by nuryagdym**

### CODE REVIEW

File: doc/symfony4.md [hunk 1]
```diff
@@ -122,6 +122,26 @@ orm:
             dir: "%kernel.root_dir%/../vendor/gedmo/doctrine-extensions/lib/Gedmo/Tree/Entity"
 ```
 
+<a name="ext-filtering"></a>
+## Filters
+
+The **softdeleteable** ORM filter also needs to be configured, so that soft deleted records are filtered when querying.
+To do so, add this filter info to your **doctrine.orm** configuration, edit **app/config/config.yml**:
```
Comment (AkenRoberts) [inline doc/symfony4.md:129]: I believe all Symfony 4+ installs have the `packages` config setup now. So this would be `/config/packages/doctrine.yaml`.

Comment (nuryagdym) [reply doc/symfony4.md:129]: you are right, those pathes need to be changed, I did not want to make too much, changes.
If you want I cant update those pathes.

Comment (AkenRoberts) [reply doc/symfony4.md:129]: Yes please!

**Comment by AkenRoberts** [comment]
You could have just updated this PR, FYI. :)

**Pull request closed without merging**